packages/nimble-components/src/utilities/unit-format/manually-translated-unit-scale/tests/volt-unit-scale.spec.ts

```diff
@@ -102,20 +102,36 @@ describe('VoltUnitScale', () => {
         }
     ] as const;
 
-    parameterizeNamedList(testCases, (spec, _name, value) => {
-        spec(`gets expected unit for ${value.number}`, () => {
-            const { scaledValue, scaledUnit: unit } = VoltUnitScale.instance.scaleNumber(value.number);
-            for (const [index, locale] of [
-                'en',
-                'fr',
-                'de',
-                'ja',
-                'zh'
-            ].entries()) {
-                expect(
-                    unit.unitFormatterFactory(locale, {}).format(scaledValue)
-                ).toEqual(value.formatted[index]!);
-            }
+    parameterizeNamedList(testCases, (spec, name, value) => {
+        spec(`gets expected unit for ${name}`, () => {
+            const { scaledValue, scaledUnit } = voltUnitScale.scaleNumber(
+                value.number
+            );
+            expect(
```

Comment:
Couple of things:

The array indices are hard-coded but type checked. With the loop TypeScript wasn't able to understand the type and a non-null assertion was needed. Unrolled TypeScript validates the index is valid for each entry, no compiler ovverrides needed.

The bigger reason is we should avoid expects in loops generally. A concrete example I ran into and changed the pattern was the following. For this snippet:
```
for (const [index, locale] of [
                'en',
                'fr',
                'de',
                'ja',
                'zh'
            ].entries()) {
                expect( // line 115
                    unit.unitFormatterFactory(locale, {}).format(scaledValue)
                ).toEqual(value.formatted[index]!);
            }
```

I had introduced a problem that failed in one of the languages. Looking just at the CI results I couldn't tell which language. Even though the test was parameterized, this wasn't in the parameterization so in the test name I couldn't tell which case failed. The stack trace doesn't help because it just says it failed on line 115 but that corresponds to 5 different expects. In this case it was pretty easy to tell what string didn't match but for numbers or booleans it's not easy.

So in general avoid expects in loops. Some alternatives:
1. Break it out into the parametrization so it is part of the test name
2. Break it into clear pre and post conditions steps. Calculate an array of values before and compare it to an array of expects. Use the jasmine matchers to help write that test:
    ```ts
    const expected = [true, true, true];
    const actual = obj.dynamicValues();
    expect(actual).toEqual(jasmine.arrayWithExactContents(expected));
    ```
3. Unroll the loop and make each expect explicit

I chose 3 for this case, 2 would have been fine too.